authnz/pom.xml

```diff
@@ -52,7 +52,7 @@
             </Bundle-Description>
             <Bundle-SymbolicName>io.stargate.auth</Bundle-SymbolicName>
             <Import-Package>org.osgi.framework</Import-Package>
-            <Export-Package>io.stargate.auth</Export-Package>
+            <Export-Package>io.stargate.auth,io.stargate.auth.entity</Export-Package>
```

Comment:
do we need to specify the inner package? 
Is the outer package `io.stargate.auth` not enough?

Reply:
we could add `.*`, but the inner package is exported because it is/will be used in the authz processor implementations.